WPThemeReview/Sniffs/ThouShallNotUse/ReservedTemplateNameSniff.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Sniffs\ThouShallNotUse;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Check if the template file is using reserved file name as a part of their name.
```

Comment:
```suggestion
 * Check if the template file is using a prefix which would cause WP to interpret it as a specialised template
 * meant to apply to only one page on the site.
```